src/main/web/templates/handlebars/content/t3.handlebars

```diff
@@ -31,8 +31,10 @@
                     <h2 class="nav-secondary__title">{{labels.view-all-content-related-to-this-topic}}:</h2>
                     <ul class="nav-secondary__list">
                         {{#if_any items datasets}}<li class="nav-secondary__item"><a href="{{uri}}/datalist">{{labels.all-data}}</a></li>{{/if_any}}
-                        <li class="nav-secondary__item"><a href="{{uri}}/publications">{{labels.all-publications}}</a>
-                        </li>
+                        <li class="nav-secondary__item"><a href="{{uri}}/publications">{{labels.all-publications}}</a></li>
+                        {{#if description.keywords}}
+                            <li class="nav-secondary__item"><a href="{{location.origin}}/news/statementsandletters/staticlist?query={{description.keywords.[0]}}">{{labels.statements-and-letters}}</a></li>
```

Comment:
Will this also take you to the Welsh version of the search page if user is in Welsh?

Reply:
It should do; the `origin` should include protocol, hostname and port if included. Will test this in develop to be sure, though, as I know testing Welsh locally can be difficult